docs/installation/teams/index.md

```diff
@@ -224,10 +224,40 @@ If you get 404, please check the ingress configuration or endpoint you configure
    If Botkube responds, send `@Botkube notifier start` to enable notifications.
    ![Teams - check bot health](assets/teams_ping.png "Teams Botkube ping")
 
+   :::warning
+   The MS Teams integration doesn't support persistence for notifications settings. As a result, when Botkube Pod restarts, or automated configuration reload happens, you need each time send `@Botkube notifier start` to re-enable notifications.
+   :::
+
 ## Remove Botkube from Kubernetes cluster
 
 Execute the following command to completely remove Botkube and related resources from your cluster.
 
 ```bash
 helm uninstall botkube --namespace botkube
 ```
+
+## Troubleshooting
+
+### Botkube doesn't respond
+
+**Symptom**
+
+When you send Botkube command on MS Teams, in Botkube logs you see:
+
+```plaintext
+Failed to parse Teams request. Authentication failed.: Unauthorized: invalid AppId passed on token" bot="MS Teams"
+```
+
+**Remedy**
```

Comment:
I think "Solution" will look better in this case:
```suggestion
**Solution**
```
WDYT?

Reply:
I'm ok with both, so I picked yours 👍